linux.md

```diff
@@ -44,10 +44,14 @@
    This installs Git Credential Manager Core.<br><br>
 
 4. Copy the following text, paste it in the terminal and hit return.<br><br>
+
    ```bash
    sudo snap install code --classic
+   sudo snap install flyctl --edge
    ```
+
    This uses Snap to install VS Code.<br><br>
```

Comment:
we need to add the package name to the language below as well (in all files)

Reply:
b06c854 

0dd96ae